### PR TITLE
Added the file "fakeusername"

### PR DESCRIPTION
**Pull request by bnehoran**
This add the file fakeusername because it is an extremely important file...

### CODE REVIEW

File: Participants/fakeusername [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Name: Fancy raccoon
+Year: 2023
+Odd Fact: I am a raccoon
```
Comment (joerup2004) [inline Participants/fakeusername:3]: Wow no way me too

**Review by joerup2004 — CHANGES_REQUESTED** [review]
You should capitalize the r in raccoon

File: Participants/fakeusername (hunk 1 above)
Comment (bnehoran) [reply Participants/fakeusername:3]: No way!

**Comment by bnehoran** [comment]
I made the changes! Please review again!

**Review by joerup2004 — APPROVED** [review]
Amazing

**Pull request merged**